server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -5,7 +5,7 @@ import * as EnumHelpers from '../core/utils/EnumHelpers';
 import { type ICardTargetSystemProperties, CardTargetSystem } from '../core/gameSystem/CardTargetSystem';
 
 export interface IMoveCardProperties extends ICardTargetSystemProperties {
-    destination?: Location;
+    destination?: Location.Discard | Location.Deck;
```

Comment:
Question: Could this also be hand, in the event we are retrieving a card from the discard pile?

Reply:
Yeah the issue is that we have two different systems that sort of cover that case (the other is ReturnToHandSystem) which is why I was filtering that out here. I have an open work item for sorting that out, so maybe for now I'll just leave this in the state it is and add a TODO since making this change makes the purpose kind of unclear as Ennis noted in their comment.